allure-cucumber6-jvm/src/main/java/io/qameta/allure/cucumber6jvm/AllureCucumber6Jvm.java

```diff
@@ -279,12 +283,19 @@ private String getHookStepUuid(final HookTestStep step) {
     }
 
     private String getHistoryId(final TestCase testCase) {
-        final String testCaseLocation = testCase.getUri().toString()
-                .substring(testCase.getUri().toString().lastIndexOf('/') + 1)
-                + ":" + testCase.getLocation().getLine();
+        final String testCaseLocation = getTestCaseUri(testCase) + ":" + testCase.getLocation().getLine();
         return md5(testCaseLocation);
     }
 
+    private String getTestCaseUri(final TestCase testCase) {
+        final String testCaseUri = testCase.getUri().toString();
+        final int workingDirIdx = testCaseUri.indexOf(CUCUMBER_WORKING_DIR);
```

Comment:
can we use `startsWith()` instead of `indexOf` ( if we use `new File("").toUri().toString()` instead of CUCUMBER_WORKING_DIR ) ?

Reply:
Done. Thanks! But I left the `CUCUMBER_WORKING_DIR` constant in place so as not to create a new object every time